static/app/components/replays/replayController.tsx

```diff
@@ -54,10 +54,10 @@ function ReplayPlayPauseBar({isCompact}: {isCompact: boolean}) {
       {!isCompact && (
         <Button
           size="xs"
-          title={t('Go back 10 seconds')}
+          title={t('Rewind 10s')}
           icon={<IconRefresh size="sm" />}
           onClick={() => setCurrentTime(currentTime - 10 * SECOND)}
-          aria-label={t('Go back 10 seconds')}
+          aria-label={t('Rewind 10 seconds')}
```

Comment:
should this be `Rewind 10s` like the `title` above?

Reply:
I get your point from a tidiness perspective for sure. I was trying to make the aria-labels as clear as possible imagining if I was using a screen reader myself. I also think the content from the tooltips could be understood just fine but I think these would be a little clearer.